Lib/ctypes/__init__.py

```diff
@@ -524,6 +524,7 @@ def WinError(code=None, descr=None):
 # functions
 
 from _ctypes import _memmove_addr, _memset_addr, _string_at_addr, _cast_addr
+from _ctypes import _memoryview_at_addr
```

Comment:
Why not merge the above imports? is it to avoid a line continuation?

Reply:
Yes.